e2e/internal/upgrade/image.go

```diff
@@ -38,17 +37,15 @@ func fetchUpgradeInfo(ctx context.Context, csp cloudprovider.Provider, toImage s
 		return upgradeInfo{}, err
 	}
 
-	measurementsURL, signatureURL, err := versionsapi.MeasurementURL(ver, csp)
+	measurementsURL, _, err := versionsapi.MeasurementURL(ver, csp)
 	if err != nil {
 		return upgradeInfo{}, err
 	}
 
 	var fetchedMeasurements measurements.M
-	_, err = fetchedMeasurements.FetchAndVerify(
+	err = fetchedMeasurements.Fetch(
```

Comment:
I don't think that we should always skip the verification even for release enterprise CLIs.

Reply:
My point is that from a end-user perspective this is strictly worse security simply dropping the verification on upgrades. Especially since we expect the user now to be able to have long lasting cluster through upgrading it. If we drop the verification on those upgrades, I don't see a point why we wouldn't also drop it from `constellation create`.

Reply:
As discussed, I missed that this is test code only.